### PR TITLE
Some mappings for 19w14b

### PR DESCRIPTION
**Pull request by liach**
Map render distance center packet and fix revenge target selection goal

Signed-off-by: liach <liach@users.noreply.github.com>

### CODE REVIEW

**Review by Runemoro — APPROVED** [review]

File: mappings/net/minecraft/entity/Entity.mapping [hunk 1]
```diff
@@ -357,6 +357,7 @@ CLASS aia net/minecraft/entity/Entity
 		ARG 3 y
 		ARG 5 z
 	METHOD h getYaw (F)F
+		ARG 1 partial
```
Comment (Gegy) [inline mappings/net/minecraft/entity/Entity.mapping]: I think the standard Fabric name is `tickDelta`

File: mappings/net/minecraft/util/math/MathHelper.mapping [hunk 2]
```diff
@@ -67,11 +69,16 @@ CLASS zr net/minecraft/util/math/MathHelper
 		ARG 0 array
 	METHOD b fastFloor (D)I
 	METHOD b floorMod (DD)D
-	METHOD b lerpClamped (DDD)D
+	METHOD b clampedLerp (DDD)D
```
Comment (Gegy) [inline mappings/net/minecraft/util/math/MathHelper.mapping:72]: Both names make sense. Any reason for this rename?

Comment (Runemoro) [reply mappings/net/minecraft/util/math/MathHelper.mapping:72]: Shouldn't we be using adjectives as prefixes everywhere?

Comment (liach) [reply mappings/net/minecraft/util/math/MathHelper.mapping:72]: This is called by `clampedLerp` in `RealmsMath`.

Moreover, the `average([D)D` method is called by `average` in `RealmsMath`, but I think `mean` may be a better name. Should I change that?

File: mappings/net/minecraft/entity/Entity.mapping (hunk 1 above)
Comment (liach) [reply mappings/net/minecraft/entity/Entity.mapping]: So should I change all the `partial` parameters in `MathHelper` to `tickDelta`

File: mappings/net/minecraft/util/math/MathHelper.mapping (hunk 2 above)
Comment (Gegy) [reply mappings/net/minecraft/util/math/MathHelper.mapping:72]: `mean` is good

File: mappings/net/minecraft/entity/Entity.mapping (hunk 1 above)
Comment (Gegy) [reply mappings/net/minecraft/entity/Entity.mapping]: I don't think so -- they may not specifically be used in the context of tick interpolation.

Comment (asiekierka) [reply mappings/net/minecraft/entity/Entity.mapping]: I'd change MathHelper's to delta over partial, and keep using tickDelta/delta

File: mappings/net/minecraft/entity/LivingEntity.mapping [hunk 3]
```diff
@@ -4,6 +4,7 @@ CLASS aij net/minecraft/entity/LivingEntity
 	FIELD aD lastLimbDistance F
 	FIELD aE limbDistance F
 	FIELD aF limbAngle F
+	FIELD aL previousYaw F
```
Comment (asiekierka) [inline mappings/net/minecraft/entity/LivingEntity.mapping]: prevYaw here, or previousHeadYaw in the other one

File: mappings/net/minecraft/util/math/MathHelper.mapping (hunk 2 above)
Comment (asiekierka) [reply mappings/net/minecraft/util/math/MathHelper.mapping:72]: meanAverage and clamedLerp, IMO

File: mappings/net/minecraft/util/math/MathHelper.mapping [hunk 4]
```diff
@@ -96,6 +113,12 @@ CLASS zr net/minecraft/util/math/MathHelper
 		ARG 4 second
 	METHOD d floor (F)I
 	METHOD d angleBetween (FF)F
+		ARG 0 first
+		ARG 1 second
+	METHOD d changeAngleGraduallyTo (FFF)F
```
Comment (asiekierka) [inline mappings/net/minecraft/util/math/MathHelper.mapping]: >Gradually

>speed

no, no, no...

File: mappings/net/minecraft/util/math/MathHelper.mapping [hunk 5]
```diff
@@ -122,7 +145,11 @@ CLASS zr net/minecraft/util/math/MathHelper
 	METHOD g isPowerOfTwo (I)Z
 	METHOD h fractionalPart (D)D
 		ARG 0 value
+	METHOD h lerpDegree (FFF)F
```
Comment (asiekierka) [inline mappings/net/minecraft/util/math/MathHelper.mapping]: lerpDegree? Sounds odd... can we do better?

File: mappings/net/minecraft/util/math/MathHelper.mapping (hunk 2 above)
Comment (Runemoro) [reply mappings/net/minecraft/util/math/MathHelper.mapping:72]: Aren't mean and average synonyms? That would be like saying "averageAverage"...

File: mappings/net/minecraft/util/math/MathHelper.mapping (hunk 5 above)
Comment (Runemoro) [reply mappings/net/minecraft/util/math/MathHelper.mapping]: `lerpAngle`? `interpolateAngle`?

File: mappings/net/minecraft/util/math/MathHelper.mapping (hunk 4 above)
Comment (liach) [reply mappings/net/minecraft/util/math/MathHelper.mapping]: will remove this mapping.

File: mappings/net/minecraft/entity/LivingEntity.mapping (hunk 3 above)
Comment (liach) [reply mappings/net/minecraft/entity/LivingEntity.mapping]: Removed this mapping. This looks to me like some delay in yaw change instead... not sure.

File: mappings/net/minecraft/util/math/MathHelper.mapping (hunk 2 above)
Comment (liach) [reply mappings/net/minecraft/util/math/MathHelper.mapping:72]: Will leave it `average`.
image

File: mappings/net/minecraft/util/math/MathHelper.mapping (hunk 5 above)
Comment (liach) [reply mappings/net/minecraft/util/math/MathHelper.mapping]: Removed. I am not professional on this and will leave it for someone else.

**Review by Gegy — APPROVED** [review]

**Review by modmuss50 — APPROVED** [review]

**Pull request merged**